src/main/java/ch/wisv/events/app/controller/ICalController.java

```diff
@@ -0,0 +1,96 @@
+package ch.wisv.events.app.controller;
+
+import biweekly.ICalVersion;
+import biweekly.ICalendar;
+import biweekly.io.text.ICalWriter;
+import ch.wisv.events.utils.ICalendarBuilder;
+import ch.wisv.events.core.service.event.EventService;
+import org.apache.commons.io.IOUtils;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+/**
+ * Copyright (c) 2016  W.I.S.V. 'Christiaan Huygens'
```

Comment:
Shouldnt this be at the top of the file? I think this is now the javadoc of this class